<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Effects</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/effect.png</normaloff>img/effect.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(213, 236, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(41, 152, 255);
font: 700 italic 20pt &quot;Segoe UI&quot;;
border-bottom-left-radius: 25%;
border-bottom-right-radius: 25%;</string>
    </property>
    <property name="text">
     <string>Effects</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_filter9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>166</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 221, 255);</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
